include/graphics.c

```diff
@@ -40,16 +46,100 @@ int windowInit() {
+
+    ultraSmallFont = TTF_OpenFont(RESOURCEDIR"Serif.ttf", 25);
+    if (!bigFont) {
+        printf("Couldn't load ultra small font! Reason: %s\n", SDL_GetError());
+        return;
+    }
+
+    SDL_Color textColor = {255, 255, 255, 255};
+
+    SDL_Rect namePos = {windowWidth-170, 40, 60, 70};
+    SDL_Surface *name = TTF_RenderText_Blended(smallFont, "TicTacToe", textColor);
+    SDL_BlitSurface(name, NULL, windowSurface, &namePos);
+    SDL_FreeSurface(name);
+
+    SDL_Rect madeByPos = {windowWidth-170, 130, 70, 80};
+    SDL_Surface *madeBy = TTF_RenderText_Blended_Wrapped(ultraSmallFont, "Made by Teufelchen1 and kosmas12", textColor, 170);
+    SDL_BlitSurface(madeBy, NULL, windowSurface, &madeByPos);
+    SDL_FreeSurface(madeBy);
+
+    SDL_Rect copyrightPos = {20, 420, 100, 60};
+    SDL_Surface *copyright = TTF_RenderText_Blended(smallFont, "Copyright 2021 and licensed under the GPL3", textColor);
+    SDL_BlitSurface(copyright, NULL, windowSurface, &copyrightPos);
+    SDL_FreeSurface(copyright);
+
+    SDL_UpdateWindowSurface(window);
+
+}
+
+void UpdateScore() {
+    SDL_Color textColor = {255, 255, 255, 255};
+
+    SDL_Rect pScorePos = {10, 0, 100, 62};
+    char *pScoreText = (char *) calloc(8, sizeof(char));
+    snprintf(pScoreText, 8, "You: %d", playerScore);
+    SDL_Surface *pScore = TTF_RenderText_Blended(ultraSmallFont, pScoreText, textColor);
+    SDL_BlitSurface(pScore, NULL, windowSurface, &pScorePos);
+    free(pScoreText);
+    SDL_FreeSurface(pScore);
+
+    SDL_Rect aiScorePos = {10, 40, 100, 20};
+    char *aiScoreText = (char *) calloc(6, sizeof(char));
+    snprintf(aiScoreText, 6, "AI: %d", aiScore);
+    SDL_Surface *aScore = TTF_RenderText_Blended(ultraSmallFont, aiScoreText, textColor);
+    SDL_BlitSurface(aScore, NULL, windowSurface, &aiScorePos);
+    free(aiScoreText);
+    SDL_FreeSurface(aScore);
+
+    SDL_Rect drawsPos = {10, 80, 200, 20};
+    char *drawsText = (char *) calloc(10, sizeof(char));
+    snprintf(drawsText, 9, "Draws: %d", draws);
+    SDL_Surface *drawsSurface = TTF_RenderText_Blended(ultraSmallFont, drawsText, textColor);
+    SDL_BlitSurface(drawsSurface, NULL, windowSurface, &drawsPos);
+    free(drawsText);
+    SDL_FreeSurface(drawsSurface);
+
+    SDL_UpdateWindowSurface(window);
+}
+
 void DrawField() {
+
     SDL_FillRect(windowSurface, NULL, SDL_MapRGB(windowSurface->format, 0, 0, 0));
-    SDL_UpdateWindowSurface(window);
+
+    SDL_Rect vert1 = {windowWidth/2-130, 40, 5, 370};
+    SDL_Rect vert2 = {windowWidth/2+40, 40, 5, 370};
+    SDL_Rect hor1 = {110, 130, 340, 5};
+    SDL_Rect hor2 = {110, 300, 340, 5};
+
+    SDL_FillRect(windowSurface, &vert1, SDL_MapRGB(windowSurface->format, 255, 255, 255));
+    SDL_FillRect(windowSurface, &vert2, SDL_MapRGB(windowSurface->format, 255, 255, 255));
+    SDL_FillRect(windowSurface, &hor1, SDL_MapRGB(windowSurface->format, 255, 255, 255));
+    SDL_FillRect(windowSurface, &hor2, SDL_MapRGB(windowSurface->format, 255, 255, 255));
```

Comment:
Is there a way in SDL to render a Surface once & keeping that for ever? That way we only need to do **one** drawing when resetting the screen. Might be good for performance and it could make the code more easily understandable later on.